scripts/build-local.py

```diff
@@ -14,6 +15,16 @@
 RED = '\033[01;31m'
 GREEN = '\033[01;32m'
 
+
+def interrupt_handler(_signum, _frame):
+    """
+    Allows script to exit immediately when Ctrl-C is pressed
+    """
+    sys.exit(130)
```

Comment:
FWIW, `os` contains more descriptive error codes
https://docs.python.org/3/library/os.html#os._exit
not sure it's required here though.

Reply:
Perhaps

```diff
diff --git a/scripts/build-local.py b/scripts/build-local.py
index 23c760c..7c33198 100755
--- a/scripts/build-local.py
+++ b/scripts/build-local.py
@@ -21,7 +21,7 @@ def interrupt_handler(_signum, _frame):
     """
     Allows script to exit immediately when Ctrl-C is pressed
     """
-    sys.exit(130)
+    sys.exit(signal.SIGINT)


 signal.signal(signal.SIGINT, interrupt_handler)
```

I chose 130 because that is what shells have the return code to when Ctrl-C is pressed for me in the past.